.storybook/preview.js

```diff
@@ -12,4 +13,15 @@ Vue.component('nuxt-link', {
   template: '<a href="#" @click.prevent="log()"><slot>NuxtLink</slot></a>',
 })
 
+Vue.prototype.$t = () => {
+  return 'Traduction'
+}
```

Comment:
Is this work in progress?
If you don't plan to setup `i18n` in storybook I propose this dummy implementation, to at least see the translation key:

```suggestion
Vue.prototype.$t = (t) => {
  return t
}
```

Reply:
We will not implement the complete i18n yeah. I like to do as stub component like for nuxt-link. Yeah i will try with the key good point